.github/workflows/push-tags.yml

```diff
@@ -43,8 +43,17 @@ jobs:
         IMAGE=gcr.io/${{ env.PROJECT_ID }}/sandbox/loadgenerator/gke:${{ steps.get_version.outputs.VERSION }}
         docker build -t $IMAGE ./src/loadgenerator
         docker push $IMAGE
+    - name: Install Hugo and npm
+      timeout-minutes: 20
+      run: sudo apt-get install hugo npm
+    - name: Install PostCSS
+      timeout-minutes: 20
+      run: sudo npm install -g autoprefixer postcss-cli postcss
+    - name: Generate Website
+      timeout-minutes: 20
+      run: hugo --source=website/ --destination=deploy/
```

Comment:
running this seemed caused a weird behavior error on the VM. It looks like it's creating infinite nested deploy folders:

```
Error: Error copying static files to /home/sanche/cloud-ops-sandbox/website/deploy: stat /home/sanche/cloud-ops-sandbox/website/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/deploy/content/en/docs/User Guide/Learn Ops Management/cloud_error_reporting.md: file name too long
```

Reply:
I'm having trouble getting this command to run. Maybe we should just commit all these files to the repo for now, and transition to docker containers later?

Reply:
Hey! I can throw some time on your calendar to walk through it and see what is going on if you prefer. It does look like that's a bug of some kind.

Reply:
Website content has been generated and added to the PR